.circleci/config.yml

```diff
@@ -561,6 +565,49 @@ jobs:
           event: fail
           template: basic_fail_1 # see https://github.com/CircleCI-Public/slack-orb/wiki#templates.
 
+  run_and_convert_notebooks_to_docs:
+    docker:
+      - image: 307629990626.dkr.ecr.us-east-1.amazonaws.com/deci/infra/circleci/runner/sg-gpu:<< pipeline.parameters.sg_docker_version >>
```

Comment:
SG docker version would point to something older...
Not sure there is something to do at the moment, and I guess we are sinners for doing it elsewehere as well.
I am just a bit worried that we would actually be using the installed version on our image rather then the updated one since notebooks usually just pip install.

Reply:
Good point. I think we can uninstall the SG and indeed rely on SG installation in the notebook itself. Or use the generic pytorch container image (But I'd prefer not to because of maintenance reasons)

Reply:
See my comment below, I think this is should kept as is https://github.com/Deci-AI/super-gradients/pull/1493#discussion_r1343988540